public/userDashboard/userProfile.html

```diff
@@ -0,0 +1,128 @@
+		<script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
+		<script src="https://code.highcharts.com/highcharts.js"></script>
+		<script src="https://code.highcharts.com/modules/data.js"></script>
+		<script src="../userDashboard/dashboard.js"></script>
+
+		<style>
+			.ad {
+				position: absolute;
+				width: 300px;
+				height: 250px;
+				border: 1px solid #ddd;
+				left: 50%;
+				transform: translateX(-50%);
+				top: 250px;
+				z-index: 10;
+			}
+			.ad .close {
+				position: absolute;
+				font-family: 'ionicons';
+				width: 20px;
+				height: 20px;
+				color: #fff;
+				background-color: #999;
+				font-size: 20px;
+				left: -20px;
+				top: -1px;
+				display: table-cell;
+				vertical-align: middle;
+				cursor: pointer;
+				text-align: center;
+			}
+		</style>
+		<script type="text/javascript">
+			$(function() {
+				$('.close').click(function() {
+					$('.ad').css('display', 'none');
+				})
+			})
+		</script>
+
+	</head>
+	<body>
+		<div class="header">
+			<div class="logo">
+				<i class="fa fa-tachometer"></i>
+				<span>Brand</span>
+			</div>
+			<a href="#" class="nav-trigger"><span></span></a>
+		</div>
+		<div class="side-nav">
+			<div class="logo">
+				<i class="fa fa-tachometer"></i>
+				<span>Brand</span>
+			</div>
+			<nav>
+				<ul>
+					<li>
+						<a href="userProfile.html" >
+							<span><i class="fa fa-user"></i></span>
+							<span>User Profile</span>
+						</a>
+					</li>
+					<li>
+						<a href="createAd.html">
+
+							<span><i class="fa fa-envelope"></i></span>
+							<span>Create An Ad</span>
+						</a>
+					</li>
+					<li class="active">
+						<a href="manageAds.html">
+							<span><i class="fa fa-bar-chart"></i></span>
+							<span>Manage Your Ads</span>
+						</a>
+					</li>
+				</ul>
+			</nav>
+		</div>
+		<div class="main-content">
+			<div class="title">
+				User Profile
+			</div>
+			<div class="main">
+				<div class="widget">
+                    <div class="title">Profile Information</div>
+                    <table class="userTable">
+                        <tr>
+                            <th>Username</th>
+                            <th>First Name</th>
+							<th>Last Name</th>
+							<th>Date of Birth</th>
+							<th>Email</th>
+							<th>Phone Number</th>
+							<th>Payment Info</th>
+                        </tr>
+                        <tr id="userInfo">
+                            <th id="displayUsername">test</th>
+                            <th id="displayFirstName">Herrick</th>
+							<th id="displayLastName">Feng</th>
+							<th id="displayDOB">12/09/1997</th>
+							<th id="displayEmail">email@gmail</th>
+							<th id="displayPhone">0424100883</th>
+							<th id="displayCard">3344</th>
+						</tr>
+					</table>
+					<form action="editUserProfile.html">
+						<input type="submit" value="Edit Profile Information" />
+					</form>
+					<div class="chart"></div>
+				</div>
+			</div>
+		</div>
+
+		
+	</body>
+</html>
```

Comment:
I'll trust that all the front-end is working fine, I'm not personally going to go through all this, we will see it physically later, and adjustments can be made if required

Reply:
The front end is functional, its just not formatted nicely but all the parts are there